virtual_rainforest/example_data/generation_scripts/elevation_example_data.py

```diff
@@ -0,0 +1,75 @@
+"""Elevation data for `vr_run` example.
+
+This code creates an example elevation map from a digital elevation model
+([SRTM](https://www2.jpl.nasa.gov/srtm/)) which is required to run the example hydrology
+model.
+
+The commented code is used to download an existing processed SRTM dataset for the SAFE
+Project area, covering the region 4°N 116°E to 5°N 117°E, see [SAFE
+wiki](https://safeproject.net/dokuwiki/safe_gis/srtm) for reference. The processed
+datafile can be downloaded from its [Zenodo record](https://zenodo.org/records/3490488).
+The dataset is then downscaled to match the required target resolution of 90m. At the
+moment this does not _actually_ align with the climate data, it is simply forced to the
+same coordinates and resolution.
+
+To save processing and to avoid adding requirements to the package, the resulting data
+is simply stored here and written to an appropriate file format.
+"""
+
+import numpy as np
+from xarray import DataArray
+
+from virtual_rainforest.example_data.generation_scripts.common import cell_displacements
+
+# # Load DEM in 30m resolution
+# original_data = requests.get(
```

Comment:
What happened here, why did you outcomment this part?

Reply:
It seemed like a lot of work to download and preprocess a fairly large file to generate 81 numbers, particularly when those numbers aren't currently meaningfully aligned to reality 😄 

At some point, this will become relevant again, but to make it reproducible and easy to re-run here and now, it seemed easier to retire the recipe and just provide the values.